src/services/email.service.js

```
@@ -0,0 +1,35 @@
+import nodemailer from 'nodemailer';
+import 'dotenv/config';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: Number(process.env.SMTP_PORT) || 587,
+  secure: false,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASS,
+  },
+});
```

Operational: the transporter is created with `secure: false` unconditionally (lines 4-12). While STARTTLS on port 587 is common, support for SMTPS (port 465) is often needed. Consider deriving `secure` from an environment variable or set `secure: process.env.SMTP_SECURE === 'true' || Number(process.env.SMTP_PORT) === 465` and/or document the expectation. Also consider calling `transporter.verify()` at server startup to fail fast on misconfiguration.